tokio/CHANGELOG.md

```diff
@@ -1,3 +1,133 @@
+# 1.22.0 (November 17, 2022)
+
+### Added
+ - process: add `Command::process_group` ([#5114])
+ - runtime: add `Handle::runtime_flavor` ([#5138])
+ - runtime: add `Handle::runtime_flavor` ([#5138])
+ - runtime: export metrics about the blocking thread pool ([#5161])
```

Comment:
runtime metrics is an unstable API, should this be under an `### Unstable` heading?